Source/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Castle.Core" version="4.0.0-alpha001" targetFramework="net40" />
+  <package id="Castle.Core" version="3.3.3" targetFramework="net40" />
```

Comment:
Castle.Core drops Silverlight support in v4 so v3 is needed for the Moq Silverlight test project.  I opened #257 for Silverlight discussion.